sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -639,7 +641,10 @@ def run_Read(self, transform_node):
     # step should be the type of value outputted by each step.  Read steps
     # automatically wrap output values in a WindowedValue wrapper, if necessary.
     # This is also necessary for proper encoding for size estimation.
-    coder = coders.WindowedValueCoder(transform._infer_output_coder())  # pylint: disable=protected-access
+    # Using a GlobalWindowCoder as a place holder instead of the default
+    # PickleCoder because GlobalWindowCoder is known coder.
```

Comment:
The window coder is never used by the runner, it only needs to be a known coder, so it doesn't matter.